packages/protocol/test/stability/reserve.ts

```diff
@@ -283,6 +284,42 @@ contract('Reserve', (accounts: string[]) => {
     })
   })
 
+  describe('addExchangeSpender & removeExchangeSpender', () => {
+    it('emits on add', async () => {
+      const addExchangeSpenderTx = await reserve.addExchangeSpender(exchangeAddress)
+
+      const addExchangeSpenderTxLogs = addExchangeSpenderTx.logs.filter(
```

Comment:
Please follow the existing convention of asserting the total length of events and the specific location of the desired event in the list